### PR TITLE
Coverage

### PR DESCRIPTION
**Pull request by jean-michelet**
Related to #371

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ function mergeParams (...params) {
 }
 
 function addRouteRateHook (pluginComponent, params, routeOptions) {
-  const hook = params.hook || defaultHook
+  const hook = params.hook
```
Comment (jean-michelet) [inline index.js:170]: It is already [set in globalParams](https://github.com/fastify/fastify-rate-limit/blob/595a627020837d5c4ad3de7a235f385f3e6af2a5/index.js#L73) and [merged with custom params](https://github.com/fastify/fastify-rate-limit/blob/595a627020837d5c4ad3de7a235f385f3e6af2a5/index.js#L130)

File: README.md [hunk 2]
```diff
@@ -435,6 +435,20 @@ The response will have the following headers if `enableDraftSpec` is `true`:
 |`ratelimit-reset`       | how many seconds must pass before the rate limit resets
 |`retry-after`           | contains the same value in time as `ratelimit-reset`
 
+### Contribute
+To run tests locally, you need a Redis instance.
+Here's a sample `docker-compose.yml` file to run Redis in your local environment:
```
Comment (Uzlopak) [inline README.md:440]: or you run `npm run redis`

Maybe dont put docker-compose.yml into .gitignore.

**Review by Uzlopak — APPROVED** [review]
RSLGTM

**Review by gurgunday — APPROVED** [review]
Great!

Thanks

**Review by Eomm — APPROVED** [review]

**Pull request merged**